CommonAssemblies.txt

```diff
@@ -1,6 +1,7 @@
 ClientUpdater.dll
 ClientUpdater.pdb
 DiscordRPC.dll
+Facepunch.Steamworks.Win64.dll
```

Comment:
Does Steamworks have 32 bit version of their dll? Or 64 bit version only available?

Reply:
yes but it seems to act as a conflict package (`Facepunch.Steamworks`) with the win64 one

currently *we* only support win64 so we can directly use the `Facepunch.Steamworks` package with minimal changes. For a full support with multiple platforms and operating systems we have to build a special NuGet package shipping the native libraries from Steam as well as modifying this C# wrapper `Facepunch.Steamworks`. It's time consuming.

Reply:
> we only support win64

What's the source on that claim? We build under either AnyCPU or 32-bit for XNA build (IIRC).

Reply:
I mean we only support Windows x64 for the Steam integration for now, due to some restrictions on Facepunch.Steamworks as well as the native Steamworks dll files. We can of course spend a lot of time to build a better NuGet package providing support for multiple architectures while it's much more time consuming

Reply:
Ah. Is it verified that the client still works in 32-bit mode, perhaps disabling steamworks integration then?